Sources/BraveWallet/WalletConstants.swift

```diff
@@ -25,6 +25,9 @@ struct WalletConstants {
   
   /// The url to Brave Help Center for Wallet.
   static let braveWalletSupportURL = URL(string: "https://support.brave.com/hc/en-us/categories/360001059151-Brave-Wallet")!
+  
+  /// The url to learn more about ENS off-chain lookups
+  static let braveWalletENSOffchainURL = URL(string: "https://github.com/brave/brave-browser/wiki/ENS-offchain-lookup")!
```

Comment:
We should probably add how to enable/disable ENS offchain look up on iOS. Right now it says open brave://settings/web3 which is not available on iOS

Reply:
Tried to make the page a bit clearer: https://github.com/brave/brave-browser/wiki/ENS-offchain-lookup